WebApplication/Startup.cs

```diff
@@ -52,6 +74,10 @@ public void ConfigureServices(IServiceCollection services)
             services.AddScoped<IReportRepository, ReportDatabaseSql>();
             services.AddScoped<ISpecializationRepository, SpecializationDatabaseSql>();
             services.AddScoped<ISurveyRepository, SurveyDatabaseSql>();
+            services.AddScoped<Backend.Repositorys.Interfaces.IAdminRepository, AdminDatabaseSql>();
```

Comment:
Just import IAdminRepository from backend, you probably already have so just delete the namespace path part